google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractResultSet.java

```diff
@@ -597,6 +612,30 @@ public boolean isNull(int columnIndex) {
       return rowData.get(columnIndex) == null;
     }
 
+    @Override
+    public <T extends AbstractMessage> T getProtoMessageInternal(int columnIndex, T message) {
```

Comment:
```suggestion
    protected <T extends AbstractMessage> T getProtoMessageInternal(int columnIndex, T message) {
```

Reply:
Thanks for the suggestion Rajat. Missed it.